src/Analysis/Ast/Impl/Analyzer/PythonAnalyzer.cs

```diff
@@ -154,8 +158,8 @@ bool IsSignificant(IPythonModule m)
 
             using (var cts = CancellationTokenSource.CreateLinkedTokenSource(_disposeToken.CancellationToken, cancellationToken)) {
                 var analysisToken = cts.Token;
- 
-                var walker = await _dependencyResolver.AddChangesAsync(key, entry, cts.Token);
+
+                var walker = await _dependencyResolver.AddChangesAsync(key, entry, cts.Token, _progress);
```

Comment:
`_dependencyResolver` is just a graph, it doesn't do any `progress` itself, so no reason to pass `IProgressService` to it. Instead, you can report the `walker.AffectedValues` at the beginning of the try block below.